src/main/java/com/hubspot/jinjava/tree/parse/Token.java

```diff
@@ -83,6 +84,37 @@ public void setRightTrimAfterEnd(boolean rightTrimAfterEnd) {
     this.rightTrimAfterEnd = rightTrimAfterEnd;
   }
 
+  /**
+   * Handle any whitespace control characters, capturing whether leading or trailing
+   * whitespace should be stripped.
+   * @param unwrapped the content of the block stripped of its delimeters
+   * @return the content stripped of any whitespace control characters.
+   */
+  protected final String handleTrim(String unwrapped) {
+    boolean parseWhitespaceControlStrictly =
+      JinjavaInterpreter.getCurrent() != null &&
+      JinjavaInterpreter
+        .getCurrent()
+        .getConfig()
+        .getLegacyOverrides()
+        .isParseWhitespaceControlStrictly();
```

Comment:
Same as https://github.com/HubSpot/jinjava/pull/902#discussion_r932519485 if you like that idea

Reply:
Definitely, thanks!